pkg/cmd/pulumi/install/install.go

```diff
@@ -147,6 +156,36 @@ func NewInstallCmd() *cobra.Command {
 	return cmd
 }
 
+// installPackagesFromProject processes packages specified in the Pulumi.yaml file
+// and installs them using similar logic to the 'pulumi package add' command
+func installPackagesFromProject(pctx *plugin.Context, proj *workspace.Project, root string) error {
+	packages := proj.GetPackageSpecs()
+	if len(packages) == 0 {
+		return nil
+	}
+
+	fmt.Println("Installing packages defined in Pulumi.yaml...")
```

Comment:
Nit: Maybe we could make this conditional on `len(packages)> 0` to avoid printing the message if there are no packages?

Reply:
never mind, there's a check above :)